pkg/workload/workload.go

```diff
@@ -59,6 +59,23 @@ type AssignmentClusterQueueState struct {
 	ClusterQueueGeneration int64
 }
 
+type InfoOptions struct {
+	excludedResourcePrefixes []string
+}
+
+type InfoOption func(*InfoOptions)
+
+var defaultOptions = InfoOptions{
+	excludedResourcePrefixes: []string{},
+}
```

Comment:
```suggestion
var defaultOptions = InfoOptions{}
```

No need to allocate an empty slice